config/locales/es.yml

```diff
@@ -1349,22 +1340,43 @@ es:
     final_tax_doc_and_unsigned_8879_have_tax_return: "%{document_type} debe estar asociado con un año fiscal"
   views:
     ctc:
+      portal:
+        home:
+          title: Â¡Gracias por presentar su declaraciÃ³n con GetCTC!
```

Comment:
oh no! all the special characters are fucked up.

Reply:
`Ã³` looks to me like we were given UTF-8 data and then re-interpreted the bytes as ISO-8859-1 aka Latin-1. See also https://en.wikipedia.org/wiki/Mojibake :D